src/setupTest.js

```diff
@@ -15,10 +15,5 @@ global.__CONFIG__ = integreatTestCmsBuildConfig
 
 // $FlowFixMe
 console.error = error => {
-  throw Error(error)
-}
-
-// $FlowFixMe
-console.warn = warn => {
-  throw Error(warn)
+  console.log(error)
 }
```

Comment:
Do we still need this at all? Why not just use the default console.error behavior?

Reply:
in my opinion we do not need it, but then I was qustioning why we had this code in first place. @maxammann do we need to redirect console.error to console.log?

Reply:
I think we can remove it, that way the app behaves exactly in the same way.